ocs_ci/templates/openshift-infra/amq/examples/templates/cluster-operator/connect-s2i-template.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
```

Comment:
Are these files from openshift-infra? Can it be cloned and only things that we change can be in our yaml directory?

Reply:
Also the directory name for templates should be templates/workloads/amq

Reply:
1) The files are gather from a downloadable link which needs authentication. The link is attached [here](https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=jboss.amq.streams)
2) I have kept the path similar to the .zip file which contains the yaml files. With this we will be able to make out which yaml files are linked to which directory.